src/ui/autocomplete.ts

```diff
@@ -50,8 +51,9 @@ const setWidth = (input: HTMLElement, options: RadarAutocompleteUIOptions) => {
     return;
   }
 
-  // if not responsive, set fixed width
+  // if not responsive, set fixed width and unset maxWidth
   input.style.width = formatCSSValue(options.width || DEFAULT_WIDTH);
+  input.style.maxWidth = 'none';
```

Comment:
might be better to just set as an empty string (or call `removeProperty`): https://stackoverflow.com/questions/2027935/how-to-remove-css-property-using-javascript

Reply:
ah that sounds like a better idea. will do that